src/Servers/Kestrel/test/InMemory.FunctionalTests/RequestTests.cs

```diff
@@ -57,6 +58,37 @@ public async Task StreamsAreNotPersistedAcrossRequests()
             }
         }
 
+        [Fact]
+        public async Task CallingCompleteDoesNotLogGreaterThanInfo()
+        {
+            var testContext = new TestServiceContext(LoggerFactory);
+
+            await using (var server = new TestServer(async context =>
+            {
+                // This will hang if 0 content length is not assumed by the server
```

Comment:
What will hang when?

Reply:
This comment was a bunch of spaghetti. Let me rethink this.

Reply:
That said, I don't think you need to post a body in this test. Kestrel will create a log warning the app didn't consume the request body. Today, I think it is info-level, but that could change. If it started to be logged as a warning, this test would start failing.

Reply:
I think I do here because I'm verifying the behavior of ConsumeAsync which I believe requires a request body to be there.